packages/components/icons/src/leading-icon/leading-icon.styles.ts

```diff
@@ -0,0 +1,156 @@
+import { designTokens } from '@commercetools-uikit/design-system';
+import { css } from '@emotion/react';
+import { type TLeadingIconProps } from './leading-icon';
+
+const sizeMap = {
+  10: designTokens.spacing50,
+  20: `calc(${designTokens.spacing50} + ${designTokens.spacing20})`,
```

Comment:
Should we perhaps add a new design token for the spacing that'll be 40px (`designTokens.spacing55`) instead of the calculation? 
@FilPob do you know if we plan in the future to use this spacing in more places?

Reply:
I think @CarlosCortizasCT already added that in another recent PR. Is that right?

Reply:
Yes, that's part of [this PR](https://github.com/commercetools/ui-kit/pull/2703).

Reply:
@chloe0592 @CarlosCortizasCT  if `spacing55` has been added recently, it is not available on `main` (screenshot is after a rebase and running `yarn install` within the past 15min).
![Screenshot 2024-01-30 at 8 27 30 AM](https://github.com/commercetools/ui-kit/assets/87667330/b4342067-6d15-40ec-b9d8-b079dd6ad811)
![Screenshot 2024-01-30 at 8 27 43 AM](https://github.com/commercetools/ui-kit/assets/87667330/e9fe0af3-d545-4bce-b2e9-90261aaa78e6)


Let me know what youd like to do here, im fine with whatever yall decide.

Reply:
Don't need to.

If this one is merged before the other one, I'll adjust it over there.